server/src/services/checkResult/checkResult.js

```diff
@@ -3,6 +3,12 @@ import { checkCode, disableCode, invalidCode } from '../../helpers/codeUtils.js'
 import { Applicant } from '../../models/Applicant.js';
 import { Transaction } from '../../models/Transaction.js';
 
+/**
+ * // TODO: Document what this is doing and why. And break it down into smaller testable functions.
+ *
```

Comment:
Searching for trulioo transactions by TransactionRecordID.
See: https://gateway-admin.trulioo.com/documentation/docs#operation/Verify
Updating the status of the applicant for whom identity or document verification has been initiated.